src/modules/rlm_imap/imap.h

```diff
@@ -0,0 +1,225 @@
+//
+//  rlm_imap
+//  
+//
+//  Created by Stephen Blackwell on 3/16/20.
+//
+
+#ifndef rlm_imap_h
```

Comment:
There's no reason to use a separate header file as the struct definitions are only use by `rlm_imap.c`.  Move all _used_ structures into `rlm_imap.c`.